app/uploaders/image_uploader.rb

```diff
@@ -0,0 +1,45 @@
+class ImageUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+
+  storage :file
+
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  def cache_dir
+    "tmp/cache/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  def extension_whitelist
+    %w(jpg jpeg gif png)
```

Comment:
__[RuboCop]__

Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `[` and `]`.


<!-- Comment by [Sider](https://sider.review) -->